### PR TITLE
MEAS-1530 new remote model for calling integration end points defined in MEAS-1731

### PR DESCRIPTION
**Pull request by thejayvm**

### CODE REVIEW

File: lib/yardstick/v2_client/grant_status.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+module Yardstick
+  module V2Client
+    class GrantStatus
+      include RemoteModel
+
+      attr_accessor :response # TODO: refactor this up so it's available to other models
+
+      resource_uri '/v2/grant_status'
```
Comment (thejayvm) [inline lib/yardstick/v2_client/grant_status.rb]: Apparently we need configure hound for this project...

**Comment by pheen** [comment]
:ship:

File: lib/yardstick/v2_client/grant_status.rb (hunk 1 above)
Comment (darkhelmet) [inline lib/yardstick/v2_client/grant_status.rb]: Copy the config from the other project.

File: lib/yardstick/v2_client/remote_model.rb [hunk 2]
```diff
@@ -40,7 +40,7 @@ def instance_uri
 
       module ClassMethods
         def find(token, id)
-          from_api(get("#{resource_uri}/#{id}", query: { token: token }))
+          from_api(get("#{resource_uri}/#{id}", query: { token: token }), token: token)
```
Comment (darkhelmet) [inline lib/yardstick/v2_client/remote_model.rb]: wat?

What is this doing? Looks like it's creating the URL with the token in the query string, and then passing the token in again?

Comment (thejayvm) [inline lib/yardstick/v2_client/remote_model.rb]: The second token hash is the "extras" param for `from_api`. It's so the token get's assigned to the instance of the class returned by find, and then can be used in other methods on the object.

File: lib/yardstick/v2_client/grant_status.rb (hunk 1 above)
Comment (thejayvm) [inline lib/yardstick/v2_client/grant_status.rb]: I don't know how.... *whine & cheese*

Comment (darkhelmet) [inline lib/yardstick/v2_client/grant_status.rb]: tapeface

**Pull request merged**